Admin/AbstractAdmin.php

```diff
@@ -192,11 +193,22 @@
     /**
      * Whether or not to persist the filters in the session.
      *
+     * NEXT_MAJOR: remove this property
+     *
      * @var bool
+     *
+     * @deprecated since 3.x, to be removed in 4.0.
      */
     protected $persistFilters = false;
 
     /**
+     * Component responsible for persisting filters.
+     *
+     * @var FilterPersisterInterface|null
+     */
+    protected $filterPersister;
```

Comment:
Ok, good thinking, but please make it `private` anyway, we're trying to change things.